src/masoniteorm/connections/SQLiteConnection.py

```diff
@@ -111,7 +113,7 @@ def query(self, query, bindings, results="*"):
             dict|None -- Returns a dictionary of results or None
         """
         query = query.replace("'?'", "?")
-        print("running query: ", self, query, bindings)
```

Comment:
this is odd but this should stay for now. It helps me see what queries are running while testing